lambda/custom/src/babybuddy/types.ts

```diff
@@ -66,6 +66,8 @@ interface CreateDiaperChange {
 interface Secret {
   apiKey: string;
   apiUrl: string;
+  cfAccessClientId: string;
+  cfAccessClientSecret: string;
```

Comment:
More TS ignorance here... assuming making this optional is possible I suspect something about this interface declaration would need to be updated?